crates/astria-sequencer-client/Cargo.toml

```diff
@@ -6,14 +6,15 @@ edition = "2021"
 [dependencies]
 astria-sequencer = { path = "../astria-sequencer" }
 
+async-trait = { workspace = true }
 borsh = { workspace = true }
 ed25519-consensus = { workspace = true }
 eyre = { workspace = true }
 reqwest = { workspace = true, features = ["json"] }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
 tendermint = { workspace = true }
-tendermint-rpc = { workspace = true, features = ["http-client"]}
+tendermint-rpc = { workspace = true, features = ["http-client", "websocket-client"]}
```

Comment:
You could also feature gate both of these by creating features like this:

```
[features]
http-client = ["tendermint-rpc/http-client"]
websocket-client = ["tendermint-rpc/websocket-client"]
```

You'd then just need to add `#[cfg(feature = "http-client")]` on the impl of `SequencerClientExt` for the http client, and similarly for the websocket client.

Reply:
Introduced features `http` and `websocket`